src/test/regress/sql/pg17.sql

```diff
@@ -574,6 +574,86 @@ DROP TABLE distributed_partitioned_table CASCADE;
 DROP TABLE local_partitioned_table CASCADE;
 -- End of Test for exclusion constraints on partitioned and distributed partitioned tables in Citus environment
 
+-- Test for ALTER TABLE SET ACCESS METHOD DEFAULT
+-- Step 1: Setup
+SET default_table_access_method = 'heap';
+
+-- Create a distributed table with a specific access method
+CREATE TABLE dist_table (id SERIAL PRIMARY KEY, data TEXT) USING heap;
+SELECT create_distributed_table('dist_table', 'id');
+
+-- Create a reference table with a specific access method
+CREATE TABLE ref_table (id SERIAL PRIMARY KEY, info TEXT) USING heap;
+SELECT create_reference_table('ref_table');
+
+-- Verify the current access method for both tables
+SELECT relname, relam
+FROM pg_class
+WHERE relname IN ('dist_table', 'ref_table');
+
+-- Step 2: Change default_table_access_method
+SET default_table_access_method = 'heap'; -- Use a valid access method
```

Comment:
Can we change this to something else, because it is already heap from before and we can't see the difference